wagtail/templatetags/wagtailcore_tags.py

```diff
@@ -93,6 +93,16 @@ def wagtail_release_notes_path():
     return "%s.html" % get_main_version(VERSION)
 
 
+@register.simple_tag
+def wagtail_feature_release_whats_new_link():
+    return "https://wagtail.org/wagtail-4-1-new-in-wagtail"
+
+
+@register.simple_tag
+def wagtail_feature_release_editor_guide_link():
+    return "https://wagtail.org/wagtail-4-1-editor-guide"
```

Comment:
Do we have a plan to point this redirect to somewhere meaningful (even if it's just temporary, e.g. https://docs.wagtail.org/en/latest/editor_manual/index.html) before the release candidate goes out?

Reply:
Yes, I believe the https://wagtail-guide-staging.torchbox.dev (the current redirect) used to be an alias for https://wagtail-guide-staging.herokuapp.com/en-latest/, but the Cloudflare proxy went down last week. Either we fix the proxy or we update the link to the current editor guide. cc @thibaudcolas

Reply:
👍 I’ve updated it to point to https://docs.wagtail.org/en/stable/editor_manual/index.html for the time being. For future reference, this is `/admin/redirects/114/` in our wagtail.org backend.